src/meta/src/rpc/mod.rs

```diff
@@ -12,19 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-mod cloud_provider;
+pub mod cloud_provider;
 pub mod ddl_controller;
 pub mod election;
-mod intercept;
+pub mod intercept;
 pub mod metrics;
-pub mod server;
-pub mod service;
+// pub mod server;
+// pub mod service;
+
+#[derive(Debug)]
+pub enum MetaStoreBackend {
+    Etcd {
+        endpoints: Vec<String>,
+        credentials: Option<(String, String)>,
+    },
+    Mem,
+}
+
+pub type ElectionClientRef = std::sync::Arc<dyn ElectionClient>;
 
 pub use election::etcd::EtcdElectionClient;
 pub use election::{ElectionClient, ElectionMember};
-pub use service::cluster_service::ClusterServiceImpl;
-pub use service::ddl_service::DdlServiceImpl;
-pub use service::heartbeat_service::HeartbeatServiceImpl;
-pub use service::hummock_service::HummockServiceImpl;
-pub use service::notification_service::NotificationServiceImpl;
-pub use service::stream_service::StreamServiceImpl;
+// pub use service::cluster_service::ClusterServiceImpl;
```

Comment:
Remove the comment?

Reply:
Yes, I will take another look and refine such minor issues later if this idea is generally ok. 🥵